docs/development/howto.rst

```diff
@@ -907,6 +907,29 @@ putting this in ``docs/conf.py`` can also help sometimes::
     warnings.showwarning = warn_with_traceback
 
 
+Providing summary info for class instances
+------------------------------------------
+
+If you want to add a method to provide some basic information about a class instance,
+you should use the Python ``__str__`` method.
+
+.. code-block:: python
+
+    class foo(object):
```

Comment:
In Python examples shouldn't use "foo" and "bar", but "spam" and "ham":
https://youtu.be/anwy2MPT5RE
I.e. class `Spam` with attribute `ham`.

`__str__` has to `return ss`